src/pep8ext_naming.py

```diff
@@ -12,7 +12,7 @@
 except ImportError:
     from flake8.util import ast, iter_child_nodes
 
-__version__ = '0.8.2'
+__version__ = '0.9.0'
```

Comment:
Sometimes to differentiate between the development version and the final version folks add something like `dev` at the end, e.g., `0.9.0dev` just so anything local on developers' machines gets updates with the released version if desired. What do you think about doing that here?

Reply:
Sure, that works for me. Apparently this needs to be spelled `0.9.0.dev0` in its normalized form.